images/krte/Dockerfile

```diff
@@ -51,57 +51,57 @@ COPY wrapper.sh /usr/local/bin/
 # Trying to remount these makes for a very noisy error block in the beginning of
 # the pod logs, so we just comment out the call to it... :shrug:
 RUN echo "Installing Packages ..." \
-        && apt-get update \
```

Comment:
This was intentionally formatted for both legibility and for categorizing packages, please don't reformat files arbitrarily.